packages/crayons-core/src/components/icon/readme.md

```diff
@@ -23,28 +30,188 @@ import ReactDOM from "react-dom";
 import { FwIcon } from "@freshworks/crayons/react";
 function App() {
   return (<div>
-  <FwIcon name="add-contact" size="18" color="green"></FwIcon>
+  <FwIcon name="add-contact" size="18" color="green" ></FwIcon>
```

Comment:
This will result in typeerror in a Typescript app. Can you please correct it ?

Reply:
Fixed